seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/rest/RestService.java

```diff
@@ -524,7 +531,7 @@ public static List<PartitionDefinition> findPartitions(DorisConfig dorisConfig,
         return tabletsMapToPartition(
                 dorisConfig,
                 be2Tablets,
-                queryPlan.getOpaquedQueryPlan(),
+                queryPlan.getOpaqued_query_plan(),
```

Comment:
Why use `getOpaqued_query_plan ` instead of `getOpaquedQueryPlan ` ?

Reply:
The QueryPlan returned by doris fe is a json format, which is directly mapped to a QueryPlan object, and the field inside is this "opaqued_query_plan"